scripts/openwrt_build_ipq40xx.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Could you in fact list that this file helps with the gl-b1300?

Reply:
Is there docs for these scripts?

Reply:
not as much as I would like.

Reply:
Looks great, thanks!